flask_crud/view.py

```diff
@@ -110,7 +110,10 @@ def get(self) -> BaseQuery:
         if not self.list_enabled:
             abort(405)
 
-        query = self.query_for_user()
+        if self._access_checks_enabled():
+            query = self.query_for_user()
```

Comment:
question: what if the model class has query_for_user defined but _access_checks_enabled=False?
as a user i would expect that it still calls query_for_user but doesn't throw an exception if not implemented, just returns model_class.query
so maybe move this check down into query_for_user?

Reply:
Sounds reasonable, will do